api/services/collectionmanager/v1alpha1/manager.pb.go

```diff
@@ -0,0 +1,963 @@
+	mi := &file_api_services_collectionmanager_v1alpha1_manager_proto_msgTypes[3]
+	if protoimpl.UnsafeEnabled && x != nil {
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		if ms.LoadMessageInfo() == nil {
+			ms.StoreMessageInfo(mi)
+		}
+		return ms
+	}
+	return mi.MessageOf(x)
+}
+
+// Deprecated: Use Collection.ProtoReflect.Descriptor instead.
+func (*Collection) Descriptor() ([]byte, []int) {
+	return file_api_services_collectionmanager_v1alpha1_manager_proto_rawDescGZIP(), []int{3}
+}
+
+func (x *Collection) GetSchemaAddress() string {
+	if x != nil {
+		return x.SchemaAddress
+	}
+	return ""
+}
+
+func (x *Collection) GetLinkedCollections() []string {
+	if x != nil {
+		return x.LinkedCollections
+	}
+	return nil
+}
+
+func (x *Collection) GetFiles() []*File {
+	if x != nil {
+		return x.Files
+	}
+	return nil
+}
+
+// File contains a regular expression for file name matching and associated
+// attributes to apply the the descriptor for matching file.
+type File struct {
+	state         protoimpl.MessageState
+	sizeCache     protoimpl.SizeCache
+	unknownFields protoimpl.UnknownFields
+
+	File       string          `protobuf:"bytes,1,opt,name=file,proto3" json:"file,omitempty"`
+	Attributes *_struct.Struct `protobuf:"bytes,2,opt,name=attributes,proto3" json:"attributes,omitempty"`
+}
+
+func (x *File) Reset() {
+	*x = File{}
+	if protoimpl.UnsafeEnabled {
+		mi := &file_api_services_collectionmanager_v1alpha1_manager_proto_msgTypes[4]
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		ms.StoreMessageInfo(mi)
+	}
+}
+
+func (x *File) String() string {
+	return protoimpl.X.MessageStringOf(x)
+}
+
+func (*File) ProtoMessage() {}
+
+func (x *File) ProtoReflect() protoreflect.Message {
+	mi := &file_api_services_collectionmanager_v1alpha1_manager_proto_msgTypes[4]
+	if protoimpl.UnsafeEnabled && x != nil {
+		ms := protoimpl.X.MessageStateOf(protoimpl.Pointer(x))
+		if ms.LoadMessageInfo() == nil {
+			ms.StoreMessageInfo(mi)
+		}
+		return ms
+	}
+	return mi.MessageOf(x)
+}
+
+// Deprecated: Use File.ProtoReflect.Descriptor instead.
+func (*File) Descriptor() ([]byte, []int) {
+	return file_api_services_collectionmanager_v1alpha1_manager_proto_rawDescGZIP(), []int{4}
+}
+
+func (x *File) GetFile() string {
+	if x != nil {
+		return x.File
+	}
+	return ""
+}
+
+func (x *File) GetAttributes() *_struct.Struct {
+	if x != nil {
+		return x.Attributes
+	}
+	return nil
+}
+
+// AuthConfig contains authorization information for connecting to a registry.
+type AuthConfig struct {
+	state         protoimpl.MessageState
+	sizeCache     protoimpl.SizeCache
+	unknownFields protoimpl.UnknownFields
+
+	Username      string `protobuf:"bytes,1,opt,name=username,proto3" json:"username,omitempty"`
+	Password      string `protobuf:"bytes,2,opt,name=password,proto3" json:"password,omitempty"`
+	ServerAddress string `protobuf:"bytes,3,opt,name=server_address,json=serverAddress,proto3" json:"server_address,omitempty"`
```

Comment:
Want to reconcile serverAddress and server_address?

Reply:
@d10n This code is generated by protobuf plugins.